### PR TITLE
silence usage message if err

### PR DESCRIPTION
**Pull request by ggarri**
## Fixed Issue(s)
#432 

## Changelog

- [ ] I thought about the changelog and included a [changelog update if required](https://github.com/consensys/quorum-key-manager/blob/master/CHANGELOG.md).

### CODE REVIEW

File: cmd/migrate.go [hunk 1]
```diff
@@ -23,10 +22,6 @@ func newMigrateCommand() *cobra.Command {
 		PreRun: func(cmd *cobra.Command, args []string) {
 			preRunBindFlags(viper.GetViper(), cmd.Flags(), "key-manager")
 		},
-		PostRun: func(cmd *cobra.Command, args []string) {
```
Comment (ggarri) [inline cmd/migrate.go:26]: PostRun is never executed if there is not Run command

File: cmd/run.go [hunk 2]
```diff
@@ -15,14 +15,16 @@ func newRunCommand() *cobra.Command {
...
+			err := runCmd(cmd, args)
+			if err != nil {
+				cmd.SilenceUsage = true
+			}
+			return err
+		},
 		PreRun: func(cmd *cobra.Command, args []string) {
 			preRunBindFlags(viper.GetViper(), cmd.Flags(), "key-manager")
 		},
-		PostRun: func(cmd *cobra.Command, args []string) {
```
Comment (ggarri) [inline cmd/run.go:22]: PostRun does not trigger when you use RunE, only with Run

File: cmd/run.go [hunk 3]
```diff
@@ -15,14 +15,16 @@ func newRunCommand() *cobra.Command {
 	runCmd := &cobra.Command{
 		Use:   "run",
 		Short: "Run application",
-		RunE:  runCmd,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := runCmd(cmd, args)
+			if err != nil {
+				cmd.SilenceUsage = true
```
Comment (darioAnongba) [inline cmd/run.go:21]: missing return err?

File: cmd/run.go [hunk 4]
```diff
@@ -15,14 +15,16 @@ func newRunCommand() *cobra.Command {
...
+			}
+			return err
```
Comment (darioAnongba) [inline cmd/run.go:23]: return nil?

**Review by darioAnongba — APPROVED** [review]

File: cmd/run.go (hunk 3 above)
Comment (ggarri) [reply cmd/run.go:21]: updated

File: cmd/run.go (hunk 4 above)
Comment (ggarri) [reply cmd/run.go:23]: updated

**Pull request merged**